src/lib/insights/client.ts

```diff
@@ -33,8 +33,9 @@ const getQueryID = (selectedHits: Hits): string => {
   }
   const queryID = queryIDs[0];
   if (typeof queryID !== 'string') {
+    const docsUrl = `https://www.algolia.com/doc/guides/getting-insights-and-analytics/search-analytics/click-through-and-conversions/how-to/send-click-and-conversion-events-with-instantsearch/js/#connect-instantsearch-with-the-insights-client-for-javascript`;
```

Comment:
should be use a short url here instead ?

Reply:
; I'm not exactly sure if this is the right URL. It's clarifying for this case, but it doesn't seem very permanent

Reply:
We should use a URL shortener for these, otherwise:

- bundle size will be impacted
- URL might disappear

Reply:
Is there a page you can think of that could be more appropriate ?

I thought about just adding the docs from the `configure` `https://www.algolia.com/doc/api-reference/widgets/configure/js/` but I didn't know how to point to the current flavour (for Angular and Vue InstantSearch), whereas we can easily add Angular/Vue/React snippets to https://www.algolia.com/doc/guides/getting-insights-and-analytics/search-analytics/click-through-and-conversions/how-to/send-click-and-conversion-events-with-instantsearch/js/#connect-instantsearch-with-the-insights-client-for-javascript

Reply:
@francoischalifour ok adding it

Reply:
done here: https://github.com/algolia/instantsearch.js/pull/4231/commits/1bb98184058d81c4f3f5ddf7b0f846cb66865e4b